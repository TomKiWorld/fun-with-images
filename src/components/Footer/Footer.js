import React from 'react';

/**
 * Site footer
 */
const Footer = () => {
  return (
    <footer className='site-footer'>
      <p>Made with <span>&hearts;</span> by <a href='https://github.com/TomKiWorld' target='_blank' rel='noopener noreferrer'>TomKiWorld</a></p>
    </footer>
  );
}

export default Footer;
