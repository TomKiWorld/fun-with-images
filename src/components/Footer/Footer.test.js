import { shallow } from 'enzyme';
import React from 'react';
import Footer from './Footer';

describe('Footer Component', () => {
  it('Expect to render Footer Component', () => {
    expect(shallow(<Footer />)).toMatchSnapshot();
  });
});
