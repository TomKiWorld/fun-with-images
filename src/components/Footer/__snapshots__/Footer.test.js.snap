// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Expect to render Footer Component 1`] = `ShallowWrapper {}`;
